versions.tf

```diff
@@ -2,8 +2,8 @@ terraform {
   required_providers {
     aws = {
       source = "hashicorp/aws"
-      version = ">= 3.19.0"
+      version = ">= 5.0.0"
     }
   }
-  required_version = ">= 0.13"
+  required_version = ">= 1.3"
```

Comment:
I am not sure about this change tbh -- I think this might force a re-install of Terraform/Terragrunt on our machines

Reply:
Had to downgrade version requirement to 1, but this works!